bucket/jdtls.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "1.10.0-202204081744",
+    "description": "Java language server",
+    "homepage": "https://projects.eclipse.org/projects/eclipse.jdt.ls",
+    "license": "EPL v2.0",
+    "depends": "python",
+    "suggest": {
+        "JDK": "java/openjdk"
+    },
+    "url": "https://download.eclipse.org/jdtls/snapshots/jdt-language-server-1.10.0-202204081744.tar.gz",
+    "hash": "da4c832d89e1e66932eb809c5760043fe1abdbefc1a4492a8022edd354c873c5",
+    "installer": {
+        "script": "scoop shim add jdtls $scoopdir\\apps\\python\\current\\python.exe $dir\\bin\\jdtls"
```

Comment:
```suggestion
        "script": "shim python $global jdtls $dir\\bin\\jdtls"
```

Reply:
This actually gives: `Can't shim 'python': couldn't find 'python'`

Reply:
Fixed!